core/model/modx/processors/resource/sort.class.php

```diff
@@ -199,6 +206,17 @@ public function sortContexts() {
         return true;
     }
 
+    public function getActiveTargetMenuindex(int $target) {
+        $resource = $this->modx->getObject('modResource', $target);
+
+        if ($resource instanceof modResource) {
+            $this->menuindex = $resource->get('menuindex');
+        	return true;
```

Comment:
As it looks like there's only one place `getActiveTargetMenuindex` is used, I'd suggest it's better code to return the value you seek (the menuindex) from the method directly, instead of `true`/`false` and setting it to a class property.

Reply:
This would still be great to fix.